engine-cdi/src/main/java/org/operaton/bpm/engine/cdi/BusinessProcess.java

```diff
@@ -481,9 +481,9 @@ public VariableMap getAndClearCachedVariableMap() {
    * Get the map of cached variables and clear the internal variable cache.
    *
    * @return the map of cached variables
-   * @deprecated use {@link #getAndClearCachedVariableMap()} instead
+   * @deprecated Use {@link #getAndClearCachedVariableMap()} instead.
    */
-  @Deprecated(forRemoval = true)
+  @Deprecated(forRemoval = true, since = "1.0.0-beta-1")
```

Comment:
General question: I was also thinking about adding "since". I would not use the "beta" suffix. The actual release since when it is deprecated is 1.0.0, and deprecations I would expect only on a minor version level. For me, "since 1.0" would be the right message. Maybe you raise that topic for discussion in the forum?

Reply:
It was decided that it must be `since = "1.0"`, see forum thread